onnxruntime/python/tools/bert/benchmark_bert.py

```diff
@@ -0,0 +1,357 @@
+    torch.set_grad_enabled(False)
+
+    for model_name in model_names:
+        config = AutoConfig.from_pretrained(model_name, torchscript=torchscript, cache_dir=cache_dir)
+        model = AutoModel.from_pretrained(model_name, config=config, cache_dir=cache_dir)
+        tokenizer = AutoTokenizer.from_pretrained(model_name, cache_dir=cache_dir)
+        max_input_size = tokenizer.max_model_input_sizes[model_name]
+        if verbose:
+            print(f"Model {model}")
+            print(f"Number of parameters {model.num_parameters()}")
+
+        for batch_size in batch_sizes:
+            if fp16:
+                model.half()
+            device = "cuda" if use_gpu else "cpu"
+            model.to(device)
+
+            for sequence_length in sequence_lengths:
+                if max_input_size is not None and sequence_length > max_input_size:
+                    continue
+
+                if verbose:
+                    print("Run PyTorch on {} with input shape {}".format(model_name, [batch_size, sequence_length]))
+
+                input_ids = torch.randint(low=0,
+                                          high=config.vocab_size - 1,
+                                          size=(batch_size, sequence_length),
+                                          dtype=torch.long,
+                                          device=device)
+                try:
+                    if torchscript:
+                        if verbose:
+                            print("Tracing model with sequence size {}".format(input_ids.shape))
+                        inference = torch.jit.trace(model, input_ids)
+                        inference(input_ids)
+                    else:
+                        inference = model
+                        inference(input_ids)
+
+                    runtimes = timeit.repeat(lambda: inference(input_ids), repeat=repeat_times, number=1)
+                    average_time = sum(runtimes) / float(len(runtimes))
+
+                    result = {
+                        "runtime": "torchscript" if torchscript else "torch",
+                        "version": torch.__version__,
+                        "device": device,
+                        "fp16": fp16,
+                        "model_name": model_name,
+                        "batch_size": batch_size,
+                        "sequence_length":sequence_length,
+                        "average_latency_ms": "{:.2f}".format(average_time * 1000),
+                        }
+
+                    print(result)
+                    results.append(result)
+                except RuntimeError as e:
+                    print("Runtime error: {}".format(e))
+                    torch.cuda.empty_cache()
+    return results
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--models",
+        required=False,
+        type=str,
+        default="all",
+        help="Pre-trained models to benchmark"
```

Comment:
Add more instructions on what is expected in this argument. e.g. model paths, each split by ','